### ISSUE TITLE
Endpoint "/api/engine/containers/{containerId}/exec" not returning "stderr"

### ISSUE THREAD

**Issue by somediego** [SaaShup/netbox-docker-agent#133]
Command works fine, but output will return "stdout" without "stderr". What make it harder to debug.

To replicate, body example:
```
{
    "cmd": [
        "ls",
        "bb"
    ]
}
```

returns: ` {"stdout":""}`

while stderr output: 
`"ls: bb: No such file or directory"`

**Comment by lvenier** [comment]
hello,

Thanks for your issue.

We can easily add the stderr in the response. By adding AttachStderr=true in the exec command.

for the same command you will get : 
```
ls: bb: No such file or directory
```

with the following json format : {"stdout":"ls: bb: No such file or directory\n"}

**Comment by fanshan** [comment]
Fixed by #134
